src/components/EmailSubscription/EmailSubscription.scss

```diff
@@ -1,3 +1,41 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
 
-/* Delete these lines when you begin development */
+.EmailSubscription {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 100%;
+
+  .EmailSubscription__searchBox {
```

Comment:
```suggestion
  .&__searchBox {
```

Reply:
Ohh, that's what you meant!

Reply:
@ilopezro - any recommendations for circumventing the `selector-class-pattern`  StyleLint error: `Classes must be in BEM form like `MyComponent__element--variant``

If we include the ampersand in the class name, it doesn't match the convention.

Reply:
I'll take a look at it!

Reply:
Yes, so you have to 
```suggestion
  &__searchBox {
  }
  &__inputButton {
  }
  &__feedbackButton {
  }
```

Reply:
But that's not how modifying classes work with ampersands... ampersands are only applicable if that element also inherits its respective parent class. Taking a look at the screenshot below, we can see the `input` and `button` elements do not inherit their parent `EmailSubscription` class, so the ampersands will not apply correctly (also tested).

Can you clarify you perspective?

![image](https://user-images.githubusercontent.com/33771623/97032291-6e5d3c80-1516-11eb-8cf7-f8ddfe8ea520.png)

Reply:
Sounds good. Then let's move away from nesting if they don't inherit parent props. Instead of it all being under the `.EmailComponent` tag let's put them outside

Reply:
A little confused by what you said there - are you talking about removing the `EmailSubscription` class/wrapper-element?

Reply:
By having everything nested within the `.EmailComponent` class, we are assuming that it's children inherit all of their props. Let's move whatever doesn't belong inside `.EmailComponent` outside of that class. So we can see a structure like 
```scss
.EmailComponet{...props}
.EmailComponent__inputButton {...props} 
```

Reply:
Okay, so just to understand you correctly - do you want me to extrapolate `searchBox` and `inputButton` into their own components? Wouldn't that be too verbose of an extrapolation - they're just HTML components.

Reply:
https://github.com/CruzHacks/cruzhacks-2021-website/blob/8c0f58a4e70459d732df92953cf211e1e620c22c/src/components/EmailSubscription/EmailSubscription.scss#L4-L41

Reply:
This is what I was doing before you suggested [nesting them](https://github.com/CruzHacks/cruzhacks-2021-website/pull/8#discussion_r506146642). So if they do not inherit any of the props, just have it go outside of the `.EmailSubscription` component and have it be it's own component. This will help with readability.

Reply:
Sorry, I misunderstood you with my last comment. It's not too verbose and works alright, but I wouldn't say it's `its own component` - it's just not nested, which is different in my perspective.